addons/web_editor/static/src/components/media_dialog/media_dialog.scss

```diff
@@ -1,114 +1,54 @@
-$min-row-height: 128;
+$-imgHeight: 120px;
 
 .modal:not(.o_legacy_dialog) .o_select_media_dialog {
-    .nav-item {
-        cursor: pointer;
-    }
-
-    .o_we_horizontal_collapse {
-        width: 0 !important;
-        padding: 0 !important;
-        border: none !important;
-    }
-
-    .o_we_transition_ease {
-        transition: all ease 0.25s;
-    }
 
     .o_we_existing_attachments {
-        min-height: $min-row-height + px;
-
-        .o_we_attachment_placeholder {
-            flex-grow: $min-row-height;
-            flex-basis: $min-row-height + px;
-        }
-
         .o_existing_attachment_cell {
-            cursor: pointer;
-            margin: 1px;
-
             &.o_we_image {
-                position: fixed !important;
+                position: fixed;
                 opacity: 0;
-                transition: opacity 0.5s;
-                transition-delay: 0.5s;
+                transition: opacity 0.5s ease 0.5s;
 
                 &.o_loaded {
-                    position: relative !important;
-                    opacity: 1;
+                    img {
+                        height: $-imgHeight;
+                    }
                 }
 
                 .o_we_media_dialog_img_wrapper {
                     @extend %o-preview-alpha-background;
-
-                    img {
-                        width: 100%;
-                    }
                 }
             }
 
             .o_existing_attachment_remove {
-                background-color: rgba(white, 0.4);
-                cursor: pointer;
-                opacity: 0;
                 transition: color 0.2s ease;
-            }
-
-            .o_existing_attachment_remove {
-                @include o-position-absolute($top: 0, $right: 0);
                 z-index: 1;
-                border-radius: 0 0 0 2px;
+                border-radius: 0 0 0 $o-we-item-border-radius;
                 &:hover {
                     color: $o-we-color-danger;
                 }
             }
 
-            .o_file_name {
-                @include o-text-overflow;
-            }
-
             &:hover {
                 .o_existing_attachment_remove {
-                    opacity: 1;
+                    opacity: 1 !important;
```

Comment:
not necessary, you can add a `opacity-100-hover` class to the button and an `opacity-trigger-hover` class to the `o_existing_attachment_cell` container